ng5/src/app/models/userService.ts

```diff
@@ -0,0 +1,16 @@
+import { User } from './user'; 
+import { HttpClient } from '@angular/common/http';
+import { environment } from '../../environments/environment';
+
+export class UserService {
+  
+  userBaseEndpoint: string = environment.WBOOK_SERVICE + '/api/v1/users';
+
+  createUser(user: User, http:HttpClient) {
+    http.post<any>(this.userBaseEndpoint, {user})
+      .subscribe(res => {
```

Comment:
the service should return the observable. The components subscribe to the result